src/index.html

```diff
@@ -9,6 +9,12 @@
   <link rel="stylesheet" href="./styles/main.scss">
 </head>
 <body>
-  <h1>Stop watch</h1>
+  <div class="container">
+    <div class="container__clock">
+      <div class="container__clock--arrow">
```

Comment:
it's not BEM modificator, it's another element of the container, so the correct name would be `container__clock-arrow`
words (name of block or element own name) in BEM classes should be split through `-`

Reply:
also a bit strange that `center` is wrapped in `arrow` - it should be siblings